src/main/java/com/synopsys/integration/alert/web/model/configuration/FieldValueModel.java

```diff
@@ -43,8 +44,9 @@ public FieldValueModel(final Collection<String> values, final boolean isSet) {
 
     public Collection<String> getValues() {
         if (null != values) {
-            return values.stream().filter(singleValue -> null != singleValue).collect(Collectors.toList());
+            return values.stream().filter(Objects::nonNull).collect(Collectors.toList());
         }
+        // FIXME should this return Set.of(); ?
```

Comment:
If isSet is true we expect values to be null right now.  The code that is in place for the UI may work correctly with Set.of() with the latest changes that were put in to the UI.